grakn-bootup/src/main/java/ai/grakn/bootup/PidRetriever.java

```diff
@@ -1,45 +0,0 @@
-/*
- * Grakn - A Distributed Semantic Database
- * Copyright (C) 2016-2018 Grakn Labs Limited
- *
- * Grakn is free software: you can redistribute it and/or modify
- * it under the terms of the GNU Affero General Public License as published by
- * the Free Software Foundation, either version 3 of the License, or
- * (at your option) any later version.
- *
- * Grakn is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License
- * along with Grakn. If not, see <http://www.gnu.org/licenses/gpl.txt>.
- */
-
-package ai.grakn.bootup;
-
-/**
- *
- * Main class invoked by bash scripting
- *
- * @author Michele Orsi
- *
- */
-public class PidRetriever extends AbstractProcessHandler {
```

Comment:
Wat?

Reply:
It doesn't seem to be in use... 

`grakn-kbms` appears to use a completely different `PidRetriever` though:
https://github.com/graknlabs/grakn-kbms/blob/c242f72378f1e8e381dfa3514ac6a32313c5e462/grakn-kbms-dist/src/main/java/ai/grakn/kbms/bootup/grakn_pid/PidRetriever.java

Maybe @lolski knows more?

Reply:
I think it _was_ used at some point, but then @lolski optimised the startup scripts and this died

Reply:
Yes, we can remove this file! We're using `ai.grakn.bootup.graknengine.grakn_pid.UnixGraknPidRetriever`. Nice catch